MyStack.cs

```diff
@@ -1,7 +1,17 @@
-﻿namespace Stack
+﻿namespace Stack;
+
+public class MyStack<T>

```

Comment:
Stack would be better name than MyStack
if that name conflicts with any other names then that's what namespaces are used for.